src/freenas/etc/ix.rc.d/ix-etc

```diff
@@ -5,12 +5,13 @@
 
 # PROVIDE: ix-etc
 # REQUIRE: earlykld
-# BEFORE: fsck
+# BEFORE: fsck smartd-daemon
 
 . /etc/rc.subr
 
 generate_etc()
 {
+	LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/midclt -t 120 waitready
```

Comment:
middlewared is already the first service to start, and boot process should wait it to be ready to continue, if its not it should be investigated why its not happening.